caddyhttp/basicauth/basicauth_test.go

```diff
@@ -70,7 +71,7 @@ func TestBasicAuth(t *testing.T) {
 		if test.result == http.StatusUnauthorized {
 			headers := rec.Header()
 			if val, ok := headers["Www-Authenticate"]; ok {
-				if got, want := val[0], "Basic realm=\"Restricted\""; got != want {
+				if got, want := val[0], "Basic realm=\"Resources\""; got != want {
```

Comment:
We still have a default for "Restricted" I think it would be good to have a test still checking for this, so we are aware of this default and any changes. Looks like the change of test has replaced the existing string and maybe adding an additional check for the new changes plus the original default value would be nice.